### ISSUE TITLE
Error-message colour on newsletter error form fails accessibility check

### ISSUE THREAD

**Issue by stevejalim** [mozilla/bedrock#11016]
## Description

The error-message markup problem noted in https://github.com/mozilla/bedrock/issues/10691 was fixed in https://github.com/mozilla/bedrock/pull/11015, but this has also shown that the hyperlink colour fails accessibility checks. (Not a regression from that PR, but a separate issue)

### Steps to reproduce

1. Go to /en-US/newsletter/recovery/ and enter an email which is not in the newsletter system - not that @example.com emails will be treated as known so don't use that.
2. View the error message that is shown:

<img width="552" alt="Screenshot 2021-12-13 at 09 59 50" src="https://user-images.githubusercontent.com/101457/145791419-1cdce1f1-2bf2-4fe9-8ebf-e8945aaed34f.png">

### Expected result

The contrast of the hyperlink on the red should pass accessibilty standards

### Actual result

It currently does not:

<img width="339" alt="Screenshot 2021-12-13 at 10 03 13" src="https://user-images.githubusercontent.com/101457/145791917-be990306-1f65-44da-bbe9-a7401e3e7b30.png">

### Environment

Nightly, MacOS

### Suggested fix

(via @alexgibson ) 
> making the hyperlink white/underlined should be enough (and removing the underline on hover/focus)

**Comment by prabhxjottsingh** [comment]
Hey, @stevejalim can you please assign me this issue, I would love to contribute to mozilla.
Thanks in advance. :)

P.S.: I have understood the issue, and would also take care of the suggestion, and will PR, as per the guidelines. :)

**Comment by prabhxjottsingh** [comment]
Sorry, @stevejalim due to some personal issues, I will not be able to solve this ISSUE soon, it may took me 2-3 weeks to get back to this issue, so if you are okay with it, then its fine, otherwise you can unassign me the issue, for other people to solve it, if you can give me some time, then I will deliver the respective to you soon.
Sorry for inconvience caused. Really wanted to contribute to this project but cannot be possible soon, I guess. :(

**Comment by stevejalim** [comment]
No worries @prabhxjottsingh  - thanks for letting us know! I hope life stuff gets easier for you! 

If anyone else picks it up before you have time, they'll assign it to themselves - so maybe unassign yourself until you are ready to start?

**Comment by prabhxjottsingh** [comment]
Yup, got it.
Unassigned myself. Thanks for understanding. :)

> No worries @prabhxjottsingh - thanks for letting us know! I hope life stuff gets easier for you!
> 
> If anyone else picks it up before you have time, they'll assign it to themselves - so maybe unassign yourself until you are ready to start?

**Comment by Ayushsunny** [comment]
hey @stevejalim Actually I was trying to fix this issue but after finding the whole codebase of bedrock and also protocol I cannot find the code where the hyperlink color for newsletter error is there but yes it's in node_modules which we can't change.
please help.

**Comment by stevejalim** [comment]
Heya @Ayushsunny - I understand what you mean, and I think it's best if we call on someone with more bedrock-frontend / protocol experience than me, to be honest.

@alexgibson when you're around again, what's the most straightforward route to restyling that link, please?

**Comment by reemhamz** [comment]
Hey @Ayushsunny, good to see you picking up a ticket again! Happy new year 💫

I took a look at the code for this issue, and the file to work on is `media/css/protocol/protocol-mozilla.scss`. We would just probably need to replace the `@includes` mixin function with another, [found in our Protocol codebase](https://github.com/mozilla/protocol/blob/main/src/assets/sass/protocol/includes/_mixins.scss#L265).

Make sure to only target the specific parent component we're targeting, as we don't want it to interfere with `.error-msg`.

Let me know if you have anymore questions. Looking forward to seeing your solution! 🦊

**Comment by reemhamz** [comment]
Hey @Ayushsunny, have you been able to start on this ticket? When it comes to things like accessibility, we want to make sure we fix issues as soon as we can. Let me know if you need anymore help or if you are choosing to opt out. 🪐 If I don't get a response by Monday, I'll pick up the ticket instead 🎂

**Comment by Ayushsunny** [comment]
Hello, @reemhamz Sorry for replying late.
Actually, I was finishing my Universities assignment 
I'll fix this within 2 or 3 days.

**Comment by reemhamz** [comment]
@Ayushsunny Thanks for letting me know! Just assign me the PR when you're done ✌🏼
